### PR TITLE
AEROGEAR-2954 iOS app icons fixed again

### PR DESCRIPTION
**Pull request by wojta**
https://issues.jboss.org/browse/AEROGEAR-2954

Icons for iOS app stopped appearing again. 

So I fixed it.

### CODE REVIEW

File: Example.iOS/Example.iOS.csproj [hunk 1]
```diff
@@ -97,14 +97,15 @@
...
     <None Include="Entitlements.plist" />
-    <None Include="Info.plist" />
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="Auth\AuthenticateOptionsProvider.cs" />
     <Compile Include="Security\SecurityCheckProvider.cs" />
     <Compile Include="Security\UnsecureAlertWarning.cs" />
     <Compile Include="Resources\SecurityResources.Designer.cs">
       <DependentUpon>SecurityResources.resx</DependentUpon>
     </Compile>
+    <None Include="Resources\Info.plist" />
```
Comment (wei-lee) [inline Example.iOS/Example.iOS.csproj]: I think this line should be removed as there is no Info.plist file in that dir.

File: Example.iOS/Example.iOS.csproj [hunk 2]
```diff
@@ -178,5 +174,33 @@
       <LastGenOutput>SecurityResources.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
+  <ItemGroup>
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Contents.json" />
```
Comment (wei-lee) [inline Example.iOS/Example.iOS.csproj]: is this file checked in?

File: Example.iOS/Example.iOS.csproj [hunk 3]
```diff
@@ -178,5 +173,33 @@
...
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-72x72%402x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-76x76%401x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-76x76%402x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-76x76%403x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-App-83.5x83.5%402x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-Small-50x50%401x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\Icon-Small-50x50%402x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcons.appiconset\ItunesArtwork%402x.png" />
+    <ImageAsset Include="Resources\Icons.xcassets\LaunchImages.launchimage\Contents.json" />
+    <ImageAsset Include="Resources\Icons.xcassets\AppIcon.appiconset\Contents.json" />
```
Comment (wei-lee) [inline Example.iOS/Example.iOS.csproj]: @wojta this line is the problem. We don't have this file in the repo

Comment (wojta) [reply Example.iOS/Example.iOS.csproj]: Noo, it's there again :( There is no such thing as this file. But it keeps adding it.

**Pull request merged**